android/src/main/java/io/radar/react/RNRadarModule.java

```diff
@@ -87,16 +87,24 @@ public void getPermissionsStatus(Promise promise) {
             return;
         }
 
-        boolean foreground = ActivityCompat.checkSelfPermission(getReactApplicationContext(), Manifest.permission.ACCESS_FINE_LOCATION) == PackageManager.PERMISSION_GRANTED;
+        Activity activity = getCurrentActivity();
```

Comment:
we should probably do a `null` check on `activity`

Reply:
Will do

Reply:
Done